src/eLife/Medium/RSS/ArticleParser.php

```diff
@@ -26,12 +26,21 @@ public function getItems()
 
     public function parseParagraph($html) : string
     {
-        // @todo more robust solution.
-        $pieces = explode('class="medium-feed-snippet">', $html);
+        // Find the start of the paragraph start tag.
```

Comment:
not much more robust, can we use http://symfony.com/doc/current/components/dom_crawler.html or similar to actually parse the HTML fragment?